improved CLI installation instructions in quickstart (#839)
akuity/kargo: docs/docs/20-quickstart.mdx

    * kind: v0.17.0
    * k3d: v5.4.9
* [Helm](https://helm.sh/docs/): These instructions were tested with v3.11.2.

> * The `kargo` CLI: To install the kargo CLI, follow these steps.

Reviewer: ```suggestion
* The `kargo` CLI: To install it, follow these steps.
```

>     1. Visit [the releases page](https://github.com/akuity/kargo/releases/latest) of the Kargo GitHub repository

Reviewer: ```suggestion
    1. Visit [the releases page](https://github.com/akuity/kargo/releases/latest) of the Kargo GitHub repository.
```

    2. Download the appropriate release package for your operating system (e.g., `kargo-linux-amd64` for Linux).

>     3. Extract the contents of the downloaded package to a directory of your choice.

Reviewer 2: There's nothing to extract. The downloads are not archives.

>     3. Extract the contents of the downloaded package to a directory of your choice.
>     4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.

Reviewer: Addressing @krancour's comment. Nothing to extract, instead instruct to move binary to a `bin` directory
```suggestion
    3. Move the binary to a directory of your choice (e.g. for Linux, consider `/usr/local/bin/`).
    4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.
```

Reviewer 2: This suggested change didn't get merged.

>     4. Add the directory containing the `kargo` executable to your system's `PATH` environment variable.

Reviewer: This is great for now! However, we should include the commands to complete these steps. That would probably be better served in its own "Installing the CLI" section of the docs.

>     After completing these steps, you should be able to run the kargo command in your terminal to interact with Kargo.

Reviewer 3: ```suggestion
    After completing these steps, you should be able to run the `kargo` command in your terminal to interact with Kargo.
```
